src/services/CasperServiceByJsonRPC.ts

```diff
@@ -351,9 +351,7 @@ export class CasperServiceByJsonRPC {
     return await this.client
       .request({
         method: 'chain_get_state_root_hash',
-        params: {
-          block_hash: blockHashBase16 || null
-        }
```

Comment:
this was an issue for a pretty long time in js-sdk. It got caught after Fraser changed RPC validation to be more restrictive :)